doc/Installation/Installation-Ubuntu-1804-Nginx.md

```diff
@@ -5,6 +5,7 @@ path: blob/master/doc/
 **Please note the minimum supported PHP version is 7.1.3**
 
 ## Install Required Packages ##
+    apt install software-properties-common
```

Comment:
Does this also need adding to `doc/Installation/Installation-Ubuntu-1804-Apache.md` and if so do you mind doing that?

Reply:
Added f3b8317f165936355a48dac706567ec1ece59b3c